debugger/ptraceLib/CMakeLists.txt

```diff
@@ -19,6 +19,8 @@ target_link_libraries(oe_ptrace ${DL_LIB} Threads::Threads)
 target_compile_options(oe_ptrace PRIVATE 
     -Wall -Werror -Wno-attributes -Wmissing-prototypes
     -m64
+    -Wno-error=conversion
```

Comment:
debugger ptraceLib is our code. It would be better to fix the code rather than suppress the warnings.